src/Servers/Kestrel/Core/src/Internal/Http/HttpProtocol.FeatureCollection.cs

```diff
@@ -338,5 +321,19 @@ Task IHttpResponseStartFeature.StartAsync(CancellationToken cancellationToken)
 
             return InitializeResponseAsync(0);
         }
+
+        void IHttpResponseBodyFeature.DisableResponseBuffering()
+        {
+        }
+
+        Task IHttpResponseBodyFeature.SendFileAsync(string path, long offset, long? count, CancellationToken cancellation)
+        {
+            throw new NotImplementedException();
```

Comment:
So how is this supposed to work? Seems like we need to polyfill all of the servers with this no?

Reply:
Yup. I'll have them all call a shared API in Http.

Reply:
Kestrel can actually do something else more clever but for this PR, status quo is probably fine (See https://github.com/aspnet/KestrelHttpServer/pull/2049 cc @benaadams)